pkgs/development/python-modules/funsor/default.nix

```diff
@@ -23,7 +28,7 @@
 buildPythonPackage rec {
   pname = "funsor";
   version = "0.4.6";
-  format = "setuptools";
+  pyproject = true;
```

Comment:
Please don't forget to add `setuptools` to `build-system`.

Reply:
It doesn't seem to require it. Should I add it nonetheless ?

Reply:
Well I agree, we should have it there (https://github.com/pyro-ppl/funsor/blob/c96e70c89aaa0bda8dc132ae1c3498a681f618b8/setup.py#L6).
I am not sure to understand why it works without it...

Reply:
I added it.